app/src/org/commcare/dalvik/application/CommCareApplication.java

```diff
@@ -1052,6 +1051,7 @@ private void unbindUserSessionService() {
                 mIsBound = false;
                 // Detach our existing connection.
                 unbindService(mConnection);
+                stopService(new Intent(this, CommCareSessionService.class));

```

Comment:
Since we are now starting the service via `startService` we need to close it down using `stopService` because I don't think unbinding it will do that for us